docs/root/configuration/http/http_filters/connect_grpc_bridge_filter.rst

```diff
@@ -0,0 +1,11 @@
+.. _config_http_filters_connect_grpc_bridge:
```

Comment:
Can you also update relevant docs here: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_protocols/grpc

And ref link back to this filter where appropriate.

Reply:
I added the bridge to the list, is that what you mean? Or should it have a section just for connect?